### DISCUSSION TITLE
Querries Related to Jupyter Notebook

### DISCUSSION THREAD

**Discussion by saleharashid01** [AammarTufail/python_ka_chilla2023#17] [Q&A]
sir, having an issue in running the string command as you execute the samosa example. The output stores in a variable but doesn't show under the cell.
![variable](https://user-images.githubusercontent.com/121869148/210857861-f42593a0-d853-4097-915e-338c4abe06d6.jpg)
![error](https://user-images.githubusercontent.com/121869148/210857897-8eb5caaa-461f-4611-bef3-8cb25bb2742e.jpg)

**Comment by Atifidrees2023** [comment]
![image](https://user-images.githubusercontent.com/121822088/210955255-97e48ae3-fd0e-404f-883c-bf3946f5fcef.png)
Dear sir my vs running code still not working and giving this instruction

**Reply by SahilJatoi744** [reply]
Apka kernal selected nh hoga VS code restart karke right end me python ka version select kary

**Comment by zohaib-qureshi** [comment]
> sir, having an issue in running the string command as you execute the samosa example. The output stores in a variable but doesn't show under the cell.
> ![variable](https://user-images.githubusercontent.com/121869148/210857861-f42593a0-d853-4097-915e-338c4abe06d6.jpg)
> ![error](https://user-images.githubusercontent.com/121869148/210857897-8eb5caaa-461f-4611-bef3-8cb25bb2742e.jpg)
> 

You have a ' . ' instead of a ' , ' after 'medah'. Maybe that's why.

**Reply by SahilJatoi744** [reply]
ap brackets hata kar string banai chaljaiga
